packages/data-stores/src/i18n/test/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import reducer from '../reducer';
+import { setLocalizedLanguageNames } from '../actions';
+
+describe( 'i18n reducer', () => {
+	describe( 'localizedLanguageNames', () => {
+		it( 'should default to no localized language names', () => {
+			const { localizedLanguageNames } = reducer( undefined, { type: '' } );
+			expect( localizedLanguageNames ).toEqual( {} );
+		} );
+
+		it( 'should set the localized language names', () => {
```

Comment:
Should we test the following scenarios as well:

* Overwriting an existing locale
* Adding new locale, but making sure existing ones are also kept